gradle.properties

```diff
@@ -1,4 +1,21 @@
 org.gradle.caching=true
 group=org.ballerinalang.sf
 version=2.2.1
-ballerinaLangVersion=2.0.0-beta.2.1
+ballerinaLangVersion=2.0.0-beta.2
+
+stdlibIoVersion=0.6.0-beta.2
+stdlibRegexVersion=0.7.0-beta.2
+stdlibOsVersion=0.8.0-beta.2
+stdlibTimeVersion=2.0.0-beta.2
+stdlibUrlVersion=1.1.0-beta.2
+stdlibTaskVersion=2.0.0-beta.2
+stdlibFileVersion=0.7.0-beta.2
+stdlibCryptoVersion=1.1.0-beta.2
+stdlibLogVersion=1.1.0-beta.2
+stdlibMimeVersion=1.1.0-beta.2
+stdlibCacheVersion=2.1.0-beta.2
+stdlibAuthVersion=1.1.0-beta.2
+stdlibJwtVersion=1.1.0-beta.2
+stdlibOAuth2Version=1.1.0-beta.2
+stdlibUuidVersion=0.10.0-beta.2
+stdlibHttpVersion=1.1.0-beta.2
```

Comment:
Better to add these in ascending order.
So it be more readable.

Reply:
Sorted